datafusion/sqllogictest/test_files/window.slt

```diff
@@ -3315,3 +3315,23 @@ SELECT
   window1 AS (ORDER BY C3)
   ORDER BY C3
   LIMIT 5
+
+# Create table with window functions that have nullable: false columns should result in correct schema during cross join
```

Comment:
these tests fail on `main`

Reply:
Note that I couldn't include a test for the `NTILE` window function because it failed. I filed a separate bug [here](https://github.com/apache/arrow-datafusion/issues/7639)